documentation/models.py

```diff
@@ -278,17 +279,24 @@ class Example(Orderable, CloneableMixin):
     app = models.URLField(blank=True, null=True, help_text='Sharing link for example app')
     image = models.ImageField(blank=True, null=True)
 
+    CODE_FROM_CODE_FIELD = 'codeFromCodeField'
+    EMBED_APP_WITH_CODE = 'embedAppWithCode'
     app_display_type_options = [
-        ('codeFromCodeField', 'Display app with code from code field above'),
-        ('embedAppWithCode', 'Embed app with code directly from code.org project')
+        (CODE_FROM_CODE_FIELD, 'Display app with code from code field above'),
+        (EMBED_APP_WITH_CODE, 'Embed app with code directly from code.org project')
     ]
 
     app_display_type = models.CharField(
         max_length=255,
         choices=app_display_type_options,
-        default=app_display_type_options[0][0],
+        default=CODE_FROM_CODE_FIELD,
         help_text='How the app and code fields for this example are rendered')
 
+    embed_app_with_code_height = models.IntegerField(
+        'Embed app with code iframe height',
+        default=310,
```

Comment:
can you help me understand this default? the method name sounds super general, but then I'd be worried if this value was chosen super-specifically for spritelab, especially since it looks like it will take a DB migration to change it.

Reply:
The values was chosen somewhat arbitrarily for what I thought looks good on spritelab, for examples that don't have a ton of code. My understanding is that at this time we are mostly expecting to use this feature for spritelab, but it would be nice to be able to use it for other features too.

Why is it a big concern that it would take a DB migration to change this field in the future? I am not currently aware of any big difficulties with doing migrations for curriculumbuilder - am I missing some context?

Reply:
OK, this is probably fine.